crates/holochain_state/src/mutations.rs

```diff
@@ -1180,21 +1180,18 @@ pub async fn copy_cached_op_to_dht(
                 let mut stmt = txn.prepare(
                     r#"
                         SELECT
-                          Action.blob AS action_blob,
-                          Entry.blob AS entry_blob
-                        FROM DhtOp
-                        JOIN Action ON DhtOp.action_hash = Action.hash
-                        LEFT JOIN Entry ON Action.entry_hash = Entry.hash
+                            Action.blob AS action_blob,
```

Comment:
When copying a sys validation dependency from the cache DB (where it landed from a network fetch) to the DHT DB, the op type that is needed for sys validation is not relevant, because a network `get` always fetches a `StoreRecord` op. Only action and entry of the record are required to construct the op that is needed to validate the warranted op.